.github/workflows/coverage.yml

```diff
@@ -0,0 +1,76 @@
+name: Coverage
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
Since we can't push directly to `main` branch, there is no point in this event here.